SECURITY.md

```diff
@@ -4,8 +4,8 @@
 
 Versions currently being supported with security updates.
 
-| Version | Supported          |
-| ------- | ------------------ |
+| Version |     Supported      |
+|---------|--------------------|
 | 4.x     | :white_check_mark: |
```

Comment:
Should we change this to 6.x?

Reply:
Or remove it and let it fall back to the security.md we have in general? 

 https://github.com/Sundsvallskommun/.github/blob/main/.github/SECURITY.md

Reply:
Yes, this is better, since it will lead to less maintenance.

Reply:
I'll remove all overriding of .github file then